### DISCUSSION TITLE
Where does LazyVim get the icons shown in neo-tree?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [LazyVim/LazyVim#5215] [Q&A] [answered]
The icons in LazyVim looks different than what I get in my own config, where I use neo-tree. I was curious about this... and wondering where they come from. I can't see anything particular in [here](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/editor.lua#L5) that would reveal this... 🤔 

Does anyone have any idea what/where the secret sauce is?

| Neo-tree default | LazyVim |
|------------|-------------|
| <img width="344" alt="image" src="https://github.com/user-attachments/assets/63859f32-6786-4936-9fae-3860344b248f" /> | <img width="336" alt="image" src="https://github.com/user-attachments/assets/671d7e0c-21ea-4c8f-8cbb-513a8d150ec6" /> |

**Comment by folke** [comment, marked as answer]
`mini.icons` instead of `nvim-devicons`

**Reply by fredrikaverpil** [reply]
Awesome, I simply copied [this part of LazyVim](https://github.com/LazyVim/LazyVim/commit/045faec0351ff9a38fda1af9b06bd7f948e84d28) from LazyVim and now I've got the same kind of icons. Many thanks!
